### PR TITLE
[fix/cart]: reset status of cart after purchase

### PR DESCRIPTION
**Pull request by elvivi90**
**Ticket:**
[fix/cart] (add link to Jira ticket)

**Description:**
There was a bug when finishing a checkout order. After a success purchase, the cart still has the items already bought.

**Solution:**
Remove items from cart in cart's reducer.
Also delete them from localStorage.

**Evidence:**
(Add images, videos, payload if required)

### CODE REVIEW

File: src/pages/cart/payment/Payment.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { processPayment } from '../../../api/api';
 import MetaData from '../../../components/util/MetaData';
 import { Route } from '../../../router/route';
 import { clearErrors, createOrder } from '../../../store/actions/orderActions';
+import {clearCart} from '../../../store/actions/cartActions';
```
Comment (fitu) [inline src/pages/cart/payment/Payment.jsx:10]: Respect spaces before and after {}.
Should be:
`import { clearCart } from '../../../store/actions/cartActions';`

File: src/store/actions/cartActions.js [hunk 2]
```diff
@@ -36,6 +37,13 @@ const removeItemFromCart = (id) => async (dispatch, getState) => {
     localStorage.setItem(StorageKeys.CART_ITEMS_KEY, JSON.stringify(getState().cart.cartItems));
 };
 
+const clearCart = () => async (dispatch, getState) => {
+    dispatch({
+        type: CLEAR_CART,
+    });
+    localStorage.setItem(StorageKeys.CART_ITEMS_KEY, JSON.stringify(getState().cart.cartItems));
```
Comment (fitu) [inline src/store/actions/cartActions.js:44]: Do we need to set cartItems from current state?
Shouldn't this be as following?
`localStorage.setItem(StorageKeys.CART_ITEMS_KEY, '');`

File: src/pages/product/components/ProductInfo.jsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ const ProductInfo = ({ product, match, showModal }) => {
 
     const addToCart = () => {
         dispatch(addItemToCart(match.params.id, quantity));
-        alert.success('Item added to cart!');
+        alert('Item added to cart!');
```
Comment (fitu) [inline src/pages/product/components/ProductInfo.jsx:36]: Is it okay **alert(...)** ? Or it should be **alert.success(...)** ?

File: src/store/actions/cartActions.js (hunk 2 above)
Comment (elvivi90) [reply src/store/actions/cartActions.js]: changed to `localStorage.setItem(StorageKeys.CART_ITEMS_KEY, '');`

File: src/pages/product/components/ProductInfo.jsx (hunk 3 above)
Comment (elvivi90) [reply src/pages/product/components/ProductInfo.jsx:36]: it throw me an error with the ".success", maybe it was a class from bootstrap that no longer exist.

File: src/pages/cart/payment/Payment.jsx (hunk 1 above)
Comment (elvivi90) [reply src/pages/cart/payment/Payment.jsx]: resolved!

**Pull request merged**